notebooks/validation_notebooks/image_validation.ipynb

```diff
@@ -0,0 +1,1651 @@
+{
```

Comment:
<p>can you come up with some metrics for the validation? Such as a pixel histogram of differences, and that two draws from SLSim noise properties become indistinguishable from the one draw from the DC2 image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>